graylog2-server/src/main/java/org/graylog2/audit/AuditEventTypes.java

```diff
@@ -38,11 +38,9 @@
     public static final String CLUSTER_CONFIGURATION_CREATE = PREFIX + "cluster_configuration:create";
     public static final String CLUSTER_CONFIGURATION_DELETE = PREFIX + "cluster_configuration:delete";
     public static final String CLUSTER_CONFIGURATION_UPDATE = PREFIX + "cluster_configuration:update";
-    public static final String CONTENT_PACK_APPLY = PREFIX + "content_pack:apply";
     public static final String CONTENT_PACK_CREATE = PREFIX + "content_pack:create";
     public static final String CONTENT_PACK_DELETE = PREFIX + "content_pack:delete";
-    public static final String CONTENT_PACK_EXPORT = PREFIX + "content_pack:export";
-    public static final String CONTENT_PACK_UPDATE = PREFIX + "content_pack:update";
+    public static final String CONTENT_PACK_DELETE_REV = PREFIX + "content_pack:delete_rev";
```

Comment:
This new event needs to be registered in the `EVENT_TYPES` constant. 

See jenkins failure:

```
Failed tests: 
  AuditCoverageTest.testAuditCoverage:83 [Check that there are no @AuditEvent annotations with unregistered event types] 
Expecting empty but was:<[public void org.graylog2.rest.resources.system.contentpacks.ContentPackResource.deleteContentPack(org.graylog2.contentpacks.model.ModelId,int)]>
  AuditCoverageTest.testAuditEventTypeFormat:116 [Check that there are no AuditEventType strings missing in the exported list] 
Expecting empty but was:<["CONTENT_PACK_DELETE_REV=server:content_pack:delete_rev"]>
```